airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -0,0 +1,339 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data"]
+
+  selector_increment:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data"]
+      
+  selector_users:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data","*"]
+
+  requester:
+    type: HttpRequester
+    url_base: "https://api.pipedrive.com/v1"
+    http_method: "GET"
+    request_parameters:
+      api_token: "{{config['api_token']}}"
+      limit: "50"
+      items: "{{ parameters.path_extractor }}"
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: "DefaultPaginator"
+      pagination_strategy:
+        type: "PageIncrement"
+        page_size: 50
+      page_token_option:
+        type: "RequestOption"
+        inject_into: "request_parameter"
+        field_name: "page"
```

Comment:
Not sure if the simple DefaultPagination will work with Pipedrive pagination response.
<img width="342" alt="image" src="https://github.com/airbytehq/airbyte/assets/5154322/a231e696-eec5-4f3c-802e-250bed6a93b0">

Reply:
@marcosmarxm ya true thanks for mentioning,
we can implement cursorpagination 
only in [lead](https://developers.pipedrive.com/docs/api/v1/Leads#getLeads) stream we cant implement cursorpagination 
can you suggest anything else for lead stream

Reply:
```
    paginator:
      type: "DefaultPaginator"
      pagination_strategy:
        type: "CursorPagination"
        cursor_value: "{{ response['additional_data']['pagination']['next_start'] }}"
      page_token_option:
        type: "RequestOption"
        inject_into: "request_parameter"
        field_name: "start"
```